src/main/java/svenhjol/charm/crafting/block/BlockLantern.java

```diff
@@ -44,6 +47,13 @@ public BlockLantern(String variant) {
         setDefaultState(blockState.getBaseState().withProperty(HANGING, false));
     }
 
+    @Nonnull
+    @Override
+    public Block setSoundType(@Nonnull SoundType sound)
```

Comment:
Does this need to be overridden?

Reply:
It does not need to, but I did it that way since basically all it does is makes the super method public.